src/data/polynomial/coeff.lean

```diff
@@ -96,6 +96,15 @@ end
 @[simp] lemma mul_coeff_zero (p q : R[X]) : coeff (p * q) 0 = coeff p 0 * coeff q 0 :=
 by simp [coeff_mul]
 
+/-- `constant_coeff p` returns the constant term of the polynomial `p`,
+  defined as `coeff p 0`. This is a ring homomorphism. -/
+@[simps] def constant_coeff : R[X] →+* R :=
+{ to_fun := λ p, coeff p 0,
+  map_one' := by rw [coeff_one, if_pos rfl],
+  map_mul' := mul_coeff_zero,
+  map_zero' := coeff_zero 0,
+  map_add' :=  λ p q, coeff_add p q 0 }
```

Comment:
Is this just `polynomial.aeval 0`?

Reply:
[polynomial.aeval](https://leanprover-community.github.io/mathlib_docs/data/polynomial/algebra_map.html#polynomial.aeval) doesn't work when `R` is noncommutative. In general evaluation of a polynomial in R[X] at an element of R isn't multiplicative when R is noncommutative.

Reply:
... I should say evaluation isn't *necessarily* multiplicative. Of course evaluation at a central element is a multiplicative, and 0 is always central.